sdk/src/main/java/com/meniga/sdk/webservices/user/ChangePassword.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2019 Meniga Iceland Inc.
+ */
+package com.meniga.sdk.webservices.user
+
+import com.meniga.sdk.webservices.requests.QueryRequestObject
+
+internal data class ChangePassword(
```

Comment:
These Kotlin classes cannot be used for managing caching in the client because they are internal. Anything we can do about that?